networks/rpc/websocket.go

```diff
@@ -68,6 +69,15 @@ func (srv *Server) WebsocketHandler(allowedOrigins []string) http.Handler {
 	return websocket.Server{
 		Handshake: wsHandshakeValidator(allowedOrigins),
 		Handler: func(conn *websocket.Conn) {
+			if atomic.LoadInt32(&srv.wsConnCount) >= MaxWebsocketConnections {
+				return
```

Comment:
@jeongkyun-oh WS client can't get any error of this?

Reply:
The client will get EOF error. Otherwise, we need to return an error response after creating a new codec, then close connection. What do you think?

Reply:
I think returing Maxconnection error message is better than `EOF`.
If possible please update the code. @jeongkyun-oh